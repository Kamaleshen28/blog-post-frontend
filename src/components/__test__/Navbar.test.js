import { render } from "@testing-library/react";
import React from "react";
import Navbar from "../Navbar";

describe('Navbar', () => {
    it('should render correctly', ()=>{
        const {asFragment} = render(<Navbar />)
        expect(asFragment()).toMatchSnapshot();
    })
})