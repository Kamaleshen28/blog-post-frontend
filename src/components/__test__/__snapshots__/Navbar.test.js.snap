// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar should render correctly 1`] = `
<DocumentFragment>
  <div
    class="navbar"
  >
    <div
      class="nav--section1"
    >
      <a>
        The Artifact
      </a>
      <a>
        Culture & Art blog
      </a>
    </div>
    <div
      class="nav--section2"
    >
      <a>
        Blog
      </a>
      <a>
        About
      </a>
      <a>
        Contact
      </a>
    </div>
  </div>
</DocumentFragment>
`;
